bin/sdk-cli.js

```diff
@@ -1,6 +1,10 @@
 #!/usr/bin/env node
 
 /** @format */
+
+// SDK bundles are for the browser. Ensure that babel preset reflects this.
+process.env.TARGET_BROWSER = 'true';
```

Comment:
Changing `process.env` doesn't feel great. However, this is effective. Given this is the first line of code in an entry-point script, it doesn't feel terrible either.

This may not be the best long-term solution. I'd love to hear other ideas.

Reply:
Might need an override for the SDK here: https://github.com/Automattic/wp-calypso/blob/0a0dbe1c32c4402b28a53cdca926c4fa5086b7bc/webpack.config.js#L212-L241

Reply:
@simison What override do you have in mind?

Reply:
Maybe my concerns here are completely invalid. It's what create-react-app does fairly liberally.

https://github.com/facebook/create-react-app/blob/30ee52cf3b2cbb6ac70999c02b1196bcaba8d4ca/packages/react-scripts/scripts/build.js#L11-L13